build/nuget/Uno.WinUI.DevServer.nuspec

```diff
@@ -0,0 +1,213 @@
+
+				<dependency id="Newtonsoft.Json" version="13.0.2" />
+				<dependency id="Microsoft.IO.RecyclableMemoryStream" version="2.3.2" />
+			</group>
+
+			<!-- net7.0-maccatalyst -->
+			<group targetFramework="net7.0-maccatalyst">
+				<dependency id="Uno.WinUI" version="1.29.0-dev.93" />
+				<dependency id="Uno.WinUI.DevServer.Messaging" version="1.29.0-dev.93" />
+
+				<dependency id="Newtonsoft.Json" version="13.0.2" />
+				<dependency id="Microsoft.IO.RecyclableMemoryStream" version="2.3.2" />
+			</group>
+
+			<!-- net7.0-macos -->
+			<group targetFramework="net7.0-macos">
+				<dependency id="Uno.WinUI" version="1.29.0-dev.93" />
+				<dependency id="Uno.WinUI.DevServer.Messaging" version="1.29.0-dev.93" />
+
+				<dependency id="Newtonsoft.Json" version="13.0.2" />
+				<dependency id="Microsoft.IO.RecyclableMemoryStream" version="2.3.2" />
+			</group>
+
+			<!-- net7.0-android -->
+			<group targetFramework="net7.0-android">
+				<dependency id="Uno.WinUI" version="1.29.0-dev.93" />
+				<dependency id="Uno.WinUI.DevServer.Messaging" version="1.29.0-dev.93" />
+
+				<dependency id="Newtonsoft.Json" version="13.0.2" />
+				<dependency id="Microsoft.IO.RecyclableMemoryStream" version="2.3.2" />
+			</group>
+
+			<!-- .NET 7.0 (Reference API) -->
+			<group targetFramework="net7.0">
+				<dependency id="Uno.WinUI" version="1.29.0-dev.93" />
+				<dependency id="Uno.WinUI.DevServer.Messaging" version="1.29.0-dev.93" />
+
+				<dependency id="Newtonsoft.Json" version="13.0.2" />
+				<dependency id="Uno.Wasm.WebSockets" version="1.1.0" />
+				<dependency id="Microsoft.IO.RecyclableMemoryStream" version="2.3.2" />
+			</group>
+
+			<!-- .NET 8.0 (Reference API) -->
+			<group targetFramework="net8.0">
+				<dependency id="Uno.WinUI" version="1.29.0-dev.93" />
+				<dependency id="Uno.WinUI.DevServer.Messaging" version="1.29.0-dev.93" />
+
+				<dependency id="Newtonsoft.Json" version="13.0.2" />
+				<dependency id="Uno.Wasm.WebSockets" version="1.1.0" />
+				<dependency id="Microsoft.IO.RecyclableMemoryStream" version="2.3.2" />
+			</group>
+		</dependencies>
+
+		<references>
+			<!-- .NET 7.0 -->
+			<group targetFramework="net7.0">
+				<reference file="Uno.UI.RemoteControl.dll" />
+			</group>
+			
+			<!-- .NET 8.0 -->
+			<group targetFramework="net8.0">
+				<reference file="Uno.UI.RemoteControl.dll" />
+			</group>
+
+			<!-- .NET 8 Android -->
+			<group targetFramework="net8.0-android">
+				<reference file="Uno.UI.RemoteControl.dll" />
+			</group>
+
+			<!-- .NET 8 iOS -->
+			<group targetFramework="net8.0-ios">
+				<reference file="Uno.UI.RemoteControl.dll" />
+			</group>
+
+			<!-- .NET 7 Android -->
+			<group targetFramework="net7.0-android">
+				<reference file="Uno.UI.RemoteControl.dll" />
+			</group>
+
+			<!-- .NET 7 iOS -->
+			<group targetFramework="net7.0-ios">
+				<reference file="Uno.UI.RemoteControl.dll" />
+			</group>
```

Comment:
Don't we need `macos` and `maccatalyst` here?

Reply:
That whole block is not useful, actually. We can let nuget do the references automatically.